kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -92,6 +101,72 @@
         @cancel="closeModal"
         @success="handleCreateSuccess()"
       />
+
+      <ClassRenameModal
+        v-show="openRenameModal"
+        :classname="classDetails.name"
```

Comment:
![image](https://github.com/user-attachments/assets/9e9f66ea-ce47-4f54-ada7-1feb32ce1e96)

Seems to be related to here (nearby?)

Reply:
> I don't think a facility will be likely to have 100+ coaches but we should certainly be prepared for it

Yes, it seems unlikely, but also worth being prepared for/refactoring for in follow up. 

> route query params so that the user's text input searches by way of the API (w/ the kind=UserKinds.COACH in the params) 

So a backend match for keyword (well, name filtering) search?